picard/ui/options/plugins.py

```diff
@@ -34,6 +34,7 @@
 from picard.ui.options import OptionsPage, register_options_page
 from picard.ui.ui_options_plugins import Ui_PluginsOptionsPage
 
+COLUMN_NAME, COLUMN_VERSION, COLUMN_ACTION, COLUMN_UPDATE = range(4)
```

Comment:
This feels like the name of column props rather than row headers.

Reply:
It was suggested by zas. Should i change these. Naming the variables is one of the most difficult task.

Reply:
That's the position of each kind of column. Not row headers.